src/lib/keys.js

```diff
@@ -34,7 +34,7 @@ Object.keys( Keys ).forEach( key => Keys[ key.toUpperCase() ] = Keys[ key ] );
 });
 
 // fn keys
-Array( 12 ).fill().forEach( ( item, index ) => Keys[ `f${ index + 1 }` ] = 112 + index );
+Array.apply(null, {length: 12}).map(Number.call, Number).forEach( ( item, index ) => Keys[ `f${ index + 1 }` ] = 112 + index );
```

Comment:
@elyobo Thank you for this contribution and clever solution! However, I found it a bit hard to understand what was happening here at first. Given that we're producing side effects anyway we might as well switch to a `for` loop from 1-12 or something like that. What do you think?